assets/css/style.css

```diff
@@ -0,0 +1,212 @@
+
+* {
+  box-sizing: border-box;
+  padding: 0;
+  margin: 0;
+  scroll-behavior: smooth;
+}
+
+body {
+  width: 375px;
+  background: #191765;
+  padding: 10px;
+}
+
+.navbar {
+  width: 375px;
+  height: 92px;
+  display: flex;
+  flex-direction: column;
+}
+
+.toolbar {
+  display: flex;
+  justify-content: space-between;
+  padding: 5px;
+}
+
+.menubar {
+  display: flex;
+  justify-content: space-between;
+  padding: 5px;
+  align-items: baseline;
+}
+
+.menubar p {
+  color: #fff;
+  font-weight: 600;
+}
+
+.menubar img {
+  padding-top: 10px;
+  margin: 5px;
+}
+
+.menubar img:hover {
+  background: rgb(150, 145, 145);
+}
+
+.headline {
+  width: 375px;
+  display: flex;
+  flex-direction: column;
+}
+
+.headtext {
+  display: flex;
+  flex-direction: column;
+  width: 327px;
+  height: 154px;
+  padding: 0;
+  align-items: center;
+  margin-top: 10px;
+}
+
+.headtext .primary-text {
+  font-family: 'Roboto', sans-serif;
+  font-style: normal;
+  font-weight: 700;
+  font-size: 30px;
+  line-height: 72px;
+  text-align: center;
+  letter-spacing: 0.37px;
+  color: #7d73fc;
+  flex: none;
+  flex-grow: 0;
+  order: 0;
+  margin: 12px 0;
```

Comment:
- [ ] Instead of using `display: flex;` multiple times, kindly create a class say

``` css
.flex {
  display: flex;
}
```
and use the class in the HTML element that needs it. This will help your code to follow the DRY rule.